src/Api/Controller/CreateTokenController.php

```diff
@@ -58,15 +58,14 @@ public function handle(ServerRequestInterface $request): ResponseInterface
 
         $identification = Arr::get($body, 'identification');
         $password = Arr::get($body, 'password');
-        $lifetime = Arr::get($body, 'lifetime', 3600);
 
         $user = $this->users->findByIdentification($identification);
 
         if (! $user || ! $user->checkPassword($password)) {
             throw new NotAuthenticatedException;
         }
 
-        $token = AccessToken::generate($user->id, $lifetime);
+        $token = AccessToken::generate($user->id);
```

Comment:
I don't want to make this too complex or introduce a feature without studying the extensibility first.

I'm just worried that it's now (temporarily) impossible to create tokens with long lifetimes from the REST API.

I don't want to introduce a `type` parameter here with some hard-coded validation check, because it's likely not how the final REST parameter would look like. Also if we allow `developer` and later add a permission, it might break again for those users.

I don't know of any integration that uses the custom `lifetime` parameter that was removed here, but if anyone was using it then it would be a pretty annoying change since there's no immediate replacement.

Or I could just go with the kind-of-backward-compatibility method, and if you use any lifetime that's different from 3600, then I make it a `session_remember` token? This might not be the intended lifetime, but I doubt anyone relied on a precise lifetime anyway (and the precise lifetime wouldn't be working because of the garbage collector issue). That's what is happening to existing tokens via the migration as well.

Reply:
What I have done is proxy the `remember` parameter from login endpoint to token endpoint. This means `remember` is now available on that endpoint for custom integrations as well.

I have added a compatibility layer for `lifetime` where any value above 3600 will have the same effect as setting `remember`